zebra/zebra_vxlan.c

```diff
@@ -4062,7 +4062,7 @@ int zebra_vxlan_dp_network_mac_add(struct interface *ifp,
 	}
 
 	/* If local MAC on a down local ES translate the network-mac-add
-	 * to a local-inactive-mac-add
+	 * to a local-active-mac-add
```

Comment:
the comment below also says `/* local_inactive */` ... should it be corrected as well, or is it actually inactive?

Reply:
It is right, that means the parameter name of function.

Reply:
In this case `local_inactive` is set to false.